latex_tables/tables_in_latex.py

```diff
@@ -16,92 +16,165 @@
-              'direct air capture',
-              'methanation',
-              'central gas boiler',
-              'decentral gas boiler',
-              'central resistive heater',
-              'decentral resistive heater',
-              'central water tank storage',
-              'decentral water tank storage', 
-              'water tank charger',
-              'decentral air-sourced heat pump',
-              'central air-sourced heat pump',
-              'decentral ground-sourced heat pump',
-              'biomass CHP capture',
-              'Fischer-Tropsch',
-              'SMR',
-              'SMR CC'
-              ]
+technologies=[
+    'onwind', 'offwind', 'solar-utility', 'solar-rooftop', 'OCGT',
+    'CCGT', 'coal', 'lignite', 'nuclear', 'hydro', 'ror', 'PHS', 
+    'central gas CHP',
+    'biomass CHP', 
+    #'central coal CHP',
+    #'biomass HOP',
+    #'biomass EOP',
+    'HVDC overhead', 'HVDC inverter pair',
+    'battery storage',
+    'battery inverter',
+    'home battery storage',
+    'home battery inverter',
+    'electrolysis',
+    'fuel cell',
+    'hydrogen storage underground',
+    'hydrogen storage tank type 1',
+    'direct air capture',
+    'methanation',
+    'central gas boiler',
+    'decentral gas boiler',
+    'central resistive heater',
+    'decentral resistive heater',
+    'central water tank storage',
+    'decentral water tank storage', 
+    'water tank charger',
+    'decentral air-sourced heat pump',
+    'central air-sourced heat pump',
+    'decentral ground-sourced heat pump',
+    'biomass CHP capture',
+    'Fischer-Tropsch',
+    'SMR',
+    'SMR CC',
+    # 'Compressed-Air-Adiabatic-bicharger',
+    # 'Compressed-Air-Adiabatic-store', 'Concrete-charger',
+    # 'Concrete-discharger', 'Concrete-store', 'Gravity-Brick-bicharger',
+    # 'Gravity-Brick-store', 'Gravity-Water-Aboveground-bicharger',
+    # 'Gravity-Water-Aboveground-store',
+    # 'Gravity-Water-Underground-bicharger',
+    # 'Gravity-Water-Underground-store', 'HighT-Molten-Salt-charger',
+    # 'HighT-Molten-Salt-discharger', 'HighT-Molten-Salt-store',
+    # 'Hydrogen-charger', 'Hydrogen-discharger', 'Hydrogen-store',
+    # 'Lead-Acid-bicharger', 'Lead-Acid-store', 'Liquid-Air-charger',
+    # 'Liquid-Air-discharger', 'Liquid-Air-store',
+    # 'Lithium-Ion-LFP-bicharger', 'Lithium-Ion-LFP-store',
+    # 'Lithium-Ion-NMC-bicharger', 'Lithium-Ion-NMC-store',
+    # 'LowT-Molten-Salt-charger', 'LowT-Molten-Salt-discharger',
+    # 'LowT-Molten-Salt-store', 'Ni-Zn-bicharger', 'Ni-Zn-store',
+    # 'Pumped-Heat-charger', 'Pumped-Heat-discharger',
+    # 'Pumped-Heat-store', 'Pumped-Storage-Hydro-bicharger',
+    # 'Pumped-Storage-Hydro-store', 'Sand-charger', 'Sand-discharger',
+    # 'Sand-store', 'Vanadium-Redox-Flow-bicharger',
+    # 'Vanadium-Redox-Flow-store', 'Zn-Air-bicharger', 'Zn-Air-store',
+    # 'Zn-Br-Flow-bicharger', 'Zn-Br-Flow-store',
+    # 'Zn-Br-Nonflow-bicharger', 'Zn-Br-Nonflow-store'

```

Comment:
intended to be commented out?

Reply:
Thought it might cause trouble somewhere else, though it does not. **Commented in now.**